app/src/main/res/values/strings.xml

```diff
@@ -43,24 +43,24 @@
     <string name="controls_add_to_playlist_title">Add To</string>
     <string name="download_path_title">Video download folder</string>
     <string name="download_path_summary">Downloaded video files are stored here</string>
-    <string name="download_path_dialog_title">Choose the download folder for video files</string>
+    <string name="download_path_dialog_title">Choose download folder for video files</string>
     <string name="download_path_audio_title">Audio download folder</string>
     <string name="download_path_audio_summary">Downloaded audio files are stored here</string>
-    <string name="download_path_audio_dialog_title">Choose the download folder for audio files</string>
+    <string name="download_path_audio_dialog_title">Choose download folder for audio files</string>
     <string name="download_choose_new_path">Change the download folders to take effect</string>
```

Comment:
I am not sure what happens here. Change appreciated.

Reply:
I'm assuming this is the prompt that asks which file you want to save downloaded audio in.

Reply:
Yes. NewPipe allows people to set different download folders for audio and video downloads.

Reply:
I meant the "Change the download folders to take effect"…?